ReSwift/Utils/AtomicBool.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AtomicBool.swift
+//  ReSwift-iOS
+//
+//  Created by 钟武 on 2018/5/2.
+//  Copyright © 2018年 Benjamin Encz. All rights reserved.
+//
+
+import Foundation
+
+/**
+ Struct is only used internally in ReSwift to implements atomic bool operation.
+ */
+internal struct AtomicBool {
```

Comment:
When you make this conform to `ExpressibleByBooleanLiteral`, you can avoid accessing the `.value` property in `Store.swift` and compare with (or assign to) `true`/`false` directly

Reply:
@DivineDominion , Emm, `ExpressibleByBooleanLiteral` only support initialized with the `Boolean` literals. So we can't compare directly. 
Maybe we can just use `.value` property to access, it's use for internal.

Reply:
Quite the contrary, == is defined for ExpressibleByBooleanLiteral, too:
https://gist.github.com/DivineDominion/12b96f625cb640dd88d4f0ebdf5f59fe

Reply:
We can't use this method as you posted. every time when we set `isDispatching`, we do make a new struct, it still not thread safe.